### PR TITLE
Improve an error message.

### PR DESCRIPTION
**Pull request by bangerth**
/rebuild

### CODE REVIEW

**Review by peterrum — APPROVED** [review]

File: source/lac/trilinos_sparse_matrix.cc [hunk 1]
```diff
@@ -2215,7 +2215,12 @@ namespace TrilinosWrappers
                                         const_cast<Epetra_CrsMatrix &>(
                                           tmp_result.trilinos_matrix()));
 #  else
-      Assert("false", ExcMessage("This function requires EpetraExt."));
+      Assert("false",
```
Comment (masterleinad) [inline source/lac/trilinos_sparse_matrix.cc:2218]:
```suggestion
      Assert("false",
```
Why is this a string?

Comment (bangerth) [reply source/lac/trilinos_sparse_matrix.cc:2218]: History. Interestingly, this makes the condition `true` :-)

**Comment by bangerth** [comment]
I fixed the condition as well -- good eyes, @masterleinad !

**Review by masterleinad — APPROVED** [review]
LGTM.

**Pull request merged**